build.gradle

```diff
@@ -812,6 +872,7 @@ if (jdkVersion == '8') {
       compile project(':iceberg-arrow')
       compile project(':iceberg-hive-metastore')
       compile project(':iceberg-spark')
+      compile project(':iceberg-hiveberg')
```

Comment:
Is this an example of a circular dependency?

Reply:
This is not. This is needed since LegacyHiveTable used in `Reader` and `SparkBatchScan` is moved to `iceberg-hiveberg` module.